### PR TITLE
Implements base work needed for theming

### PR DESCRIPTION
**Pull request by julaberry**
Standardization of the following css variables: 

Borders
- a5e-border-radius-standard

Colours
- a5e-color-primary
- a5e-color-error
- a5e-color-warning
- a5e-color-disabled
- a5e-color-text-dark
- a5e-color-text-light
- a5e-color-text-medium
- a5e-color-background-light

Text Sizes
- a5e-text-size-xxs
- a5e-text-size-xs
- a5e-text-size-sm
- a5e-text-size-md
- a5e-text-size-lg
- a5e-text-size-xl
- a5e-text-size-xxl
- a5e-text-size-xxxl
- a5e-text-size-4xl

Transitions
- a5e-transition-standard

### CODE REVIEW

File: src/apps/chat/body/CardBody.svelte
Comment (NekroDarkmoon) [inline src/apps/chat/body/CardBody.svelte:1]: Note: This needs to be tested to see if it actually works.

File: src/apps/chat/body/PromptButton.svelte [hunk 1]
```diff
@@ -77,19 +74,19 @@
         box-shadow: none;
 
         &:hover .die {
-            color: var(--hover-color, $color-dark-text);
+            color: var(--hover-color, var(--a5e-color-text-dark));
```
Comment (NekroDarkmoon) [inline src/apps/chat/body/PromptButton.svelte:77]: This doesn't work. `color: var(--hover-color, --a5e-color-text-dark)`

File: src/apps/components/ItemInnerWrapper.svelte [hunk 2]
```diff
@@ -569,10 +569,10 @@
     }
 
     .active {
-        color: var(--icon-color-active, $color-primary);
+        color: var(--icon-color-active, var(--a5e-color-primary));
```
Comment (NekroDarkmoon) [inline src/apps/components/ItemInnerWrapper.svelte:572]: Same as before. Doesn't need a nested var.

File: src/apps/components/ItemInnerWrapper.svelte [hunk 3]
```diff
@@ -569,10 +569,10 @@
...
 
         &:hover {
-            color: var(--icon-color-active, $color-primary);
+            color: var(--icon-color-active, var(--a5e-color-primary));
```
Comment (NekroDarkmoon) [inline src/apps/components/ItemInnerWrapper.svelte:575]: Same as before. Doesn't need a nested var.

File: src/apps/components/actorSidebar/RestButton.svelte [hunk 4]
```diff
@@ -61,17 +58,17 @@
...
+            transition: var(--a5e-transition-standard);
         }
 
         &:hover {
             background-color: darken(#dddace, 2.5%);
 
             .rest__icon {
-                color: lighten($color-dark-text, 35%);
+                color: #lighten(var(--a5e-color-text-dark), 35%);
             }
```
Comment (NekroDarkmoon) [inline src/apps/components/actorSidebar/RestButton.svelte:72]: Note: Need to test if this works.

File: src/apps/components/actorUtilityBar/CreateMenu.svelte [hunk 5]
```diff
@@ -52,16 +47,14 @@
         --tjs-icon-button-background-selected="none"
         --tjs-icon-button-text-shadow-hover="none"
         --tjs-icon-button-text-shadow-focus="none"
-        --tjs-icon-button-transition="all 0.15s ease-in-out"
+        --tjs-icon-button-transition="var(--a5e-transition-standard)"
         --tjs-icon-button-diameter="1.1rem"
```
Comment (NekroDarkmoon) [inline src/apps/components/actorUtilityBar/CreateMenu.svelte:51]: Note: Need to test if this works.

File: src/apps/components/effectChanges/ChangeConfiguration.svelte [hunk 6]
```diff
@@ -38,11 +38,11 @@
         --padding="0 3px"
         --item-padding="0.25rem"
         --group-item-padding-left="1rem"
-        --border-radius="3px"
-        --font-size="0.833rem"
+        --border-radius="var(--a5e-border-radius-standard)"
+        --font-size="var(--a5e-text-size-sm)"
```
Comment (NekroDarkmoon) [inline src/apps/components/effectChanges/ChangeConfiguration.svelte:42]: Need to test if these work.

File: src/apps/components/pages/EffectsEffectTab.svelte [hunk 7]
```diff
@@ -98,7 +95,7 @@
         --tjs-icon-button-background-selected="none"
         --tjs-icon-button-text-shadow-hover="none"
         --tjs-icon-button-text-shadow-focus="none"
-        --tjs-icon-button-transition="$standard-transition"
+        --tjs-icon-button-transition="var(--a5e-transition-standard)-standard)"
```
Comment (NekroDarkmoon) [inline src/apps/components/pages/EffectsEffectTab.svelte:98]: this seems to have fucked up.

**Review by NekroDarkmoon — CHANGES_REQUESTED** [review]
Some things need to be fixed while others need validation

File: src/apps/components/pages/EffectsEffectTab.svelte (hunk 7 above)
Comment (julaberry) [reply src/apps/components/pages/EffectsEffectTab.svelte]: Fixed

**Review by NekroDarkmoon — APPROVED** [review]

**Pull request merged**